public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -28,9 +28,11 @@ import { expandRecordingRules } from './language_utils';
 import { getQueryHints } from './query_hints';
 import { getOriginalMetricName, renderTemplate, transform } from './result_transformer';
 import {
+  ExemplarTraceIDDestination,
```

Comment:
nit pick, not sure if this can be change, but following the rest of the formatting, I think this should be `ExemplarTraceIdDestination`.

Reply:
agree